index.html

```diff
@@ -482,7 +489,7 @@ <h2>Sample Code</h2>
           sendCropTarget(cropTarget);
 
           function sendCropTarget(cropTarget) {
-            // Can send the crop-target to another document in this browsing context
+            // Can send the crop-target to another document in this tab
```

Comment:
Heads up. I intentionally use "tab" here because it's a comment of a mock function within a sample usage demo code. I therefore think clarity trumps accuracy in this context.